src/components/breadcrumb/Breadcrumb.tsx

```diff
@@ -9,54 +9,62 @@
  */
 
 import {ReactNode} from "react";
-import Breadcrumb from 'react-bootstrap/Breadcrumb';
+import Breadcrumb from "react-bootstrap/Breadcrumb";
 
 /**
  * Props for the BreadcrumbItem
  * @param {title} is to get the title of the Breadcrumb Item.(can be string or JSX Element)
  * @param {path} is to get the path on which url should be redirected on clicking of particular Breadcrumb Item.
  * @param {isActive} is to get the status of the Breadcrumb Item whether its active or not.
+ * @param {dataqa} quality engineering testing field
  */
 export type BreadcrumbItem = {
     title: JSX.Element | string;
     path: string;
     isActive?: boolean;
-}
+    dataqa?: string;
+};
 
 /**
  * Props for the BreadcrumbItems
  * @param {breadcrumbItems} is to get the array of Breadcrumb Elements to be displayed in the Breadcrumb.
  * @param {className} is to get a custom class to be applied on Breadcrumb or else default class is applied.
+ * @param {dataqa} quality engineering testing field
  * @param {onClick} is to get a custom handler to be applied on click of Breadcrumb Element.
  */
 export type BreadcrumbItems = {
-    breadcrumbItems:Array<BreadcrumbItem>;
-    className?:string;
-    onClick?:(event?: React.MouseEvent<HTMLElement>)=>void;
-}
+    breadcrumbItems: Array<BreadcrumbItem>;
+    className?: string;
+    dataqa?: string;
+    onClick?: (event?: React.MouseEvent<HTMLElement>) => void;
+};
 
-const getBreadcrumbItems=(items:Array<BreadcrumbItem>,onClick?:(event?: React.MouseEvent<HTMLElement>)=>void):ReactNode=>{
-    return items.map((item,index) => {
+const getBreadcrumbItems = (
+    items: Array<BreadcrumbItem>,
+    onClick?: (event?: React.MouseEvent<HTMLElement>) => void,
+): ReactNode => {
+    return items.map((item, index) => {
         return (
-        <Breadcrumb.Item key={index} href={item.path} active={item?.isActive} onClick={(event?:React.MouseEvent<HTMLElement>)=>{onClick&&onClick(event)}}>
-            {
-                <span>
-                    {item.title}
-                </span>
-            }
-        </Breadcrumb.Item>
+            <div data-qa={item?.dataqa}>
+            <Breadcrumb.Item
+                key={index}
+                href={item.path}
+                active={item?.isActive}
+                onClick={(event?: React.MouseEvent<HTMLElement>) => {
+                    onClick && onClick(event);
+                }}
+            >
+                {<span>{item.title}</span>}
+            </Breadcrumb.Item>
+            </div>
```

Comment:
```suggestion
          
            <Breadcrumb.Item
                key={index}
                data-qa={item?.dataqa}
                href={item.path}
                active={item?.isActive}
                onClick={(event?: React.MouseEvent<HTMLElement>) => {
                    onClick && onClick(event);
                }}
            >
                {<span>{item.title}</span>}
            </Breadcrumb.Item>
            
```

Reply:
Good but probably not possible. Breadcrumb item is coming from react-bootstrap so I think that's why not able to pass data-qa here.

Reply:
Yes @ShitalSD , also Breadcrumb Item is not a prop, so providing that will through TS errors.